src/Gateways/Requests/Subscriptions/SubscriptionRequest.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
The code in this file seems to be indented using spaces and not tabs. This needs to be fixed.